package.json

```diff
@@ -141,6 +147,7 @@
     "ship": "yarn surge",
     "theme": "npx gulp less",
     "watch": "node ./scripts/watch.js",
+    "snapshot": "react-scripts test --updateSnapshot",
```

Comment:
If we're gonna roll out snapshot tests, my vote is to make sure this is well disseminated (maybe updated readme or wiki page), or else this is workflow is gonna fail pretty frequently. 

I think making it well known when a dev should or shouldnt be running the 'yarn snapshot' command, would be an important first step.. since this is what likely updates our snapshot tests to a new state.